privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -97,6 +97,12 @@
 SERVERPOOL_ROUNDS = 2
 # The number of seconds a non-responding server is removed from the server pool
 SERVERPOOL_SKIP = 30
+# The number of seconds that ldap3 waits if no server is left in the pool, before
+# starting the next round
+pooling_loop_timeout = get_app_config_value("PI_LDAP_POOLING_LOOP_TIMEOUT", 10)
+log.info("Setting system wide POOLING_LOOP_TIMEOUT to {0!s}.".format(pooling_loop_timeout))
```

Comment:
Do we get this Info on each request or only when the server starts?

Reply:
imho this happens during loading of this python module - thus only once at starttime.